ls-ecaps.c

```diff
@@ -68,12 +68,12 @@ cap_ltr(struct device *d, int where)
   snoop = get_conf_word(d, where + PCI_LTR_MAX_SNOOP);
   scale = cap_ltr_scale((snoop >> PCI_LTR_SCALE_SHIFT) & PCI_LTR_SCALE_MASK);
   printf("\t\tMax snoop latency: %lldns\n",
-	 ((unsigned long long)snoop & PCI_LTR_VALUE_MASK) * scale);
+	 ((u64)snoop & PCI_LTR_VALUE_MASK) * scale);
```

Comment:
You should change the format string, too (also at other places).

Reply:
This is done in other commit _lspci: Define PCI_U64_FMT_U format for printing u64_.

Reply:
Ah, OK. Thanks!